cdc/kv/shared_client.go

```diff
@@ -815,7 +815,7 @@ func (s *SharedClient) logSlowRegions(ctx context.Context) error {
 			return ctx.Err()
 		case <-ticker.C:
 		}
-		log.Info("event feed starts to check locked regions",
```

Comment:
There shouldn't be too many such logs.

Reply:
It depends on the number of changefeed, if there hundreds of changefeed, there will be a lot of this log.